fe/fe-core/src/main/java/com/starrocks/catalog/Function.java

```diff
@@ -179,6 +179,10 @@ public String dbName() {
     public Type getReturnType() {
         return retType;
     }
+    
+    public void setReturnType(Type t) {
```

Comment:
Return type should be determinate when function construct

Reply:
fixed